CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Added
+- Add vector tile ADR [#723](https://github.com/open-apparel-registry/open-apparel-registry/pull/723)
```

Comment:
Consider linking to the ADR itself

Reply:
Good suggestion -- I'm going to leave this link as is since custom has been to link to the PR.